Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -650,6 +650,10 @@
 #define ENABLE_VIDEO_USES_ELEMENT_FULLSCREEN 1
 #endif
 
+#if HAVE(UIKIT_WEBKIT_INTERNALS)
+#define HAVE_PIP_CONTROLLER 1
```

Comment:
This should be in PlatformHave, not PlatformEnable since it has "HAVE" in the name.